### PR TITLE
[Doc]Refine README.md

### PR DESCRIPTION
**Pull request by Jennifer88huang**
Refine typos and language issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,61 +3,52 @@
...
    - job *node_metrics*: all the machines of the pulsar cluster.
 
-You can checkout the [example prometheus config](prometheus/cluster.yml.template) for
-how to configure your prometheus server to collect the metrics of a Pulsar cluster.
+How to configure your prometheus server to collect the metrics of a Pulsar cluster, refer to [example prometheus config](prometheus/cluster.yml.template).
 
 ## Usage
 
-Use this Grafana Dashboard on ...
+Use this Grafana Dashboard on ... <!--on what? please provide details.-->
```
Comment (sijie) [inline README.md]: > 

Use this Grafana Dashboard on a Standalone Cluster. It is breaking into the section `... a Standalone Cluster`.

Comment (Jennifer88huang) [reply README.md]: ok， i'll refine this soon.

Comment (Jennifer88huang) [inline README.md]:
```suggestion
Use this Grafana Dashboard on a standalone cluster.
```

File: README.md [hunk 2]
```diff
@@ -3,61 +3,52 @@
...
 
 ### ... a Standalone Cluster
```
Comment (Jennifer88huang) [inline README.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (Jennifer88huang) [inline README.md]:
```suggestion
Use this Grafana Dashboard on a standalone cluster.
```

File: README.md (hunk 2 above)
Comment (Jennifer88huang) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -3,61 +3,51 @@
...
+How to configure your prometheus server to collect the metrics of a Pulsar cluster, refer to [example prometheus config](prometheus/cluster.yml.template).
 
 ## Usage
 
-Use this Grafana Dashboard on ...
-
-### ... a Standalone Cluster
+Use this Grafana Dashboard on a standalone cluster.
 
+<!--incomplete info-->
```
Comment (Jennifer88huang) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -3,61 +3,50 @@
...
 
 ## Prometheus
 
-In order to make sure this dashboard can display the metrics correctly, you have to
-configure your Prometheus server to collect metrics from Pulsar correctly.
+To display the metrics correctly with this dashboard, configure your Prometheus server to collect metrics from Pulsar correctly.
 
-1. Make sure your prometheus service attach an extra label - `cluster`. The cluster name
-   should be aligned with the `PULSAR_CLUSTER` you provided to the grafana dashboard
+1. Attach your prometheus service to an extra label - `cluster`. The cluster name is aligned with the `PULSAR_CLUSTER` name you have provided to the grafana dashboard.
```
Comment (sijie) [inline README.md]: > Attach your prometheus service to an extra label - `cluster`.

I am not sure if this is correct.  The original sentence is to describe - configure your prometheus service to attach an extra label - `cluster` to the metrics collected by prometheus.

Comment (Jennifer88huang) [reply README.md]: Sorry, I misunderstand the original sentence. Could we refine it as follows? 
"Configure Prometheus service, and make sure your Prometheus service attaches an extra label `cluster` to the metrics collected from Pulsar cluster."

Comment (sijie) [reply README.md]: +1

Comment (Jennifer88huang) [inline README.md]:
```suggestion
1. Configure Prometheus service, and make sure your Prometheus service attaches an extra label `cluster` to the metrics collected from Pulsar cluster. The cluster name is aligned with the `PULSAR_CLUSTER` name you have provided to the grafana dashboard.
```

**Comment by Jennifer88huang** [comment]
@sijie @jiazhai i've committed the refinement in the two files. 
If there is no other issue, please approve, and then we can merge it.

**Pull request merged**